electron_app/AuthLoginModule.js

```diff
@@ -0,0 +1,99 @@
+const electronOauth2 = require('electron-oauth2');
+const constants = require('./helpers/enviroment').constants;
+
+class LoginGoogle {
+  static setWindows(win) {
```

Comment:
Not sure what `setWindows()` is doing, or why it's `windows` and not `window`. Maybe it's more like `createOauth2Window()`?